apps/builddao/data.json

```diff
@@ -1,3 +1,3 @@
 {
-  "builddao.testnet": {}
+  "buildhub.near": {}
```

Comment:
I don't think we should change this, because it's configured for testnet deployment

Reply:
I think this is just a side effect of bos-workspace, and it switches between testnet and mainnet depending on dev command that is run...

@saadiqbal-dev can you add data.json to gitignore please?